FetchMigration/Dockerfile

```diff
@@ -14,5 +16,9 @@ COPY python/*.py .
 # update PATH
 ENV PATH=/root/.local:$PATH
 
+# Write custom DP config
+RUN echo "ssl: false" > $DATA_PREPPER_PATH/config/data-prepper-config.yaml
+RUN echo "metricRegistries: [Prometheus]" >> $DATA_PREPPER_PATH/config/data-prepper-config.yaml
+
 # Include the -u flag to have stdout logged
-ENTRYPOINT python -u ./fetch_orchestrator.py $DATA_PREPPER_PATH $FM_CODE_PATH/input.yaml https://localhost:4900
+ENTRYPOINT python -u ./fetch_orchestrator.py $DATA_PREPPER_PATH $FM_CODE_PATH/input.yaml http://localhost:4900
```

Comment:
Do we have a limitation with only using http for data prepper?

Reply:
Not a limitation, but there's no need to use SSL and auth for the Data Prepper endpoint since it is only accessed within the Fetch Migration container for metrics information (and shutdown)